_scripts/instruction-widget/install.js

```diff
@@ -94,26 +96,31 @@ module.exports = function(context) {
   debian_install = function() {
     template = "debian";
 
-    if (context.distro == "ubuntu" && context.version == 16.04) {
-      context.above_4 = false;
-      context.xenial = true;
-      if (context.webserver == "apache") {
-        context.package = "python-letsencrypt-apache";
-      }
+    // Debian Jessie
+    context.base_command = "certbot";
+    context.cron_included = true;
+    if (context.webserver == "apache") {
+      context.package = "python-certbot-apache";
     } else {
-      // Debian Jessie, Ubuntu 16.10, or newer
-      context.base_command = "certbot";
-      context.cron_included = true;
-      if (context.webserver == "apache") {
-        context.package = "python-certbot-apache";
-      } else {
-        context.package = "certbot"
-      }
-      // Debian Jessie backports.
-      if (context.distro == "debian" && context.version == 8) {
-        context.backports_flag = "-t jessie-backports";
-      }
+      context.package = "certbot"
+    }
+    // Debian Jessie backports.
+    if (context.version == 8) {
+      context.backports_flag = "-t jessie-backports";
     }
+    
+  }
+
+  ubuntu_install = function() {
+    template = "ubuntu";
+
+    context.package = "certbot"
+    if (context.webserver == "apache") {
+      context.package = "python-letsencrypt-apache";
```

Comment:
You want `python-certbot-apache`, otherwise, they're going to install the `letsencrypt` version from Ubuntu.